send/atlassian_mu_process.pl

```diff
@@ -100,9 +100,10 @@ sub help {
 my $ourGroups = fetchOurGroups();
 # Data from Atlassian into hash
 my $theirUsers = fetchTheirUsers();
-my $theirGroups = fetchTheirGroups();
 
+addPrefixToDeactivatedUsers(); # REMOVE THIS LINE AFTER PREFIX INITIALIZATION
```

Comment:
```suggestion
my $DEACTIVATED_PREFIX_INIT=0;

if ($DEACTIVATED_PREFIX_INIT) {
    addPrefixToDeactivatedUsers();
}
```

+ update DEPLOYMENT NOTE accordingly (`scripts needs to be run once with DEACTIVATED_PREFIX_INIT=1`)